integration-tests/scaffolding/rootHooks.ts

```diff
@@ -1,19 +1,29 @@
 import { ExtrinsicHelper } from "./extrinsicHelpers";
 import { createKeys } from "./apiConnection";
-import { devAccounts } from "./helpers";
+import { devAccounts, rococoAccounts } from "./helpers";
 
 export let EXISTENTIAL_DEPOSIT: bigint;
 
 exports.mochaHooks = {
     async beforeAll() {
         await ExtrinsicHelper.initialize();
-        for (const uri of ["//Alice", "//Bob", "//Charlie", "//Dave", "//Eve", "//Ferdie"]) {
-            devAccounts.push({
-                uri,
-                keys: createKeys(uri),
-            });
+        let default_source;
+        if (process.env.CHAIN_ENVIRONMENT == "rococo") {
+            const seed_phrase = "Place holder for seed phrase";
```

Comment:
So as to not expose seed phrases holding token, we may want to use AWS Secrets Manager to store the seed phrase for the funding source account.

Reply:
Could this just be an environment variable?

Reply:
It can be if we would prefer to simplify the process. My thinking was that multiple environments could be added (local / testnet / mainnet) for storing funding accounts as well as any future distinct aspects to each env.

If we use an environment variable we would still want to store the seed phrase values secretly. Possibly github actions has a solution?

Reply:
I believe most secrets engines (certainly AWS; I think GH Actions, too) can inject the secret as an environment variable. While I'm not generally in favor of this (there's still a risk of exposure, if someone gains access to the pod while it's running), it seems to be the way we're currently doing things. The alternative is to have the code retrieve secrets via the secret engine's API at runtime, but that seems like overkill for e2e tests.

Reply:
A risk for exposure may also be if the env variable is leaked into actions output when injecting the variable. We would need to make sure that doesn't happen. My initial thought was to have the secret pulled at runtime, although I could be wrong about the leaking of environment variable to output. 

Preventing a leak becomes more important if we shift to using a funding account that creates token rather than simply holds and transfers it.

I think these are great options to investigate. I think this should probably be pulled into into a separate issue that creates a manual workflow to kick off e2e tests against Rococo and also investigates injecting the secret as an environment variable vs. pulling from a secret engine at runtime? @JoeCap08055

Reply:
I just checked, it does NOT leak the secrets into the output. The environment variable injected secrets would probably suffice